ghcide 0.7.4.0 breaks hls-explicit-imports-plugin 0.1.0.0

Ironically explicitly importing from `import Development.IDE.GHC.Compat` might have avoided the problem.

```
src/Ide/Plugin/ExplicitImports.hs:219:10: error:
    Ambiguous occurrence ‘isQualifiedImport’
    It could refer to
       either ‘Development.IDE.GHC.Compat.isQualifiedImport’,
              imported from ‘Development.IDE.GHC.Compat’ at src/Ide/Plugin/ExplicitImports.hs:29:1-33
           or ‘Ide.Plugin.ExplicitImports.isQualifiedImport’,
              defined at src/Ide/Plugin/ExplicitImports.hs:255:1
    |
219 |   | not (isQualifiedImport imp),
    |          ^^^^^^^^^^^^^^^^^
```

--- a/plugins/hls-explicit-imports-plugin/hls-explicit-imports-plugin.cabal
+++ b/plugins/hls-explicit-imports-plugin/hls-explicit-imports-plugin.cabal
@@ -1,6 +1,6 @@
 cabal-version: 2.2
 name:          hls-explicit-imports-plugin
-version:       0.1.0.0
+version:       0.1.0.1
 synopsis:      Explicit imports plugin for Haskell Language Server
 license:       Apache-2.0
 license-file:  LICENSE
@@ -22,7 +22,7 @@ library
                ,    haskell-lsp-types
                ,    hls-plugin-api
                ,    ghc
-               ,    ghcide
+               ,    ghcide     >= 0.7.4
                ,    shake
                ,    text
                ,    unordered-containers

